### PR TITLE
Add... a few more folks.

### PR DESCRIPTION
**Pull request by froboy**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -31,12 +31,25 @@ <h1><a href="https://midwestopensourcealliance.org/">MOSA</a></h1>
...
+            <li><a href="https://a11ytalks.com/">A11yTalks</a> - member since 2024</li>
+            <li><a href="https://www.drupalasheville.com/">Drupal Asheville</a> - member since 2024</li>
+            <li><a href="https://www.decoupleddays.com/">Decoupled Days</a> - member since 2024</li>
+            <li><a href="https://webcamp.stanford.edu/">Stanford WebCamp</a> - member since 2024</li>
+            <li><a href="https://pnwdrupalsummit.org/">Pacific Northwest Drupal Summit</a> - member since 2024</li>
+            <li><a href="https://www.drupalcampnj.org/">Drupal NJ</a> - member since 2024</li>
+            <li><a href="https://www.drupaldiversity.com/">Drupal Diversity & Inclusion</a> - member since 2024</li>
+        </ul>
+
+        <p>In addition, we are helping support initiatives that improve the event experience for both speakers and attendees, especially in areas that promote inclusivity and accessibility. We are currently supporting the <a href="https://opencollective.com/drupal-recording-initiative">Drupal Recording Initiative</a> and the <a href="https://github.com/MidCamp/live-captioning">Live Captioning Initiative</a>.</p>
```
Comment (kthull) [inline index.html:52]: In addition, we support meetups and initiatives ...

File: index.html [hunk 2]
```diff
@@ -31,12 +31,25 @@ <h1><a href="https://midwestopensourcealliance.org/">MOSA</a></h1>
...
-
+        <p>Midwest Open Source Alliance promotes the education, support and development of Drupal and related open source software within the Midwest Drupal community (and beyond). We are an Illinois nonprofit and were granted <a href="https://www.charitynavigator.org/index.cfm?bay=search.profile&ein=831560285">federal 501(c)(3)</a> tax exempt organization status in 2018 (EIN 83-1560285).</p>
+
+        <p>We are fiscal sponsor for:
+            
+        <ul>
+            <li><a href="https://www.midcamp.org/">Midwest Area Drupal Camp (MidCamp)</a> - founding member</li>
+            <li><a href="http://tcdrupal.org/">Twin Cities Drupal Camp</a> - founding member</li>
+            <li>Midwest Drupal Summit - founding member</li>
+            <li><a href="https://www.meetup.com/drupalchicago/">Drupal Chicago Meetup group</a></li>
```
Comment (kthull) [inline index.html:42]: This feels out of place in a list of member organizations, so I'd say to drop it into the paragraph below

**Review by kthull — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (froboy) [reply index.html:42]:
```suggestion
```

File: index.html (hunk 1 above)
Comment (froboy) [reply index.html:52]:
```suggestion
        <p>In addition, we support meetups and initiatives that improve the event experience for both speakers and attendees, especially in areas that promote inclusivity and accessibility. We are currently supporting:</p>
        <ul>
          <li><a href="https://opencollective.com/drupal-recording-initiative">Drupal Recording Initiative</a></li>
          <li><a href="https://github.com/MidCamp/live-captioning">Live Captioning Initiative</a></li>
          <li><a href="https://www.meetup.com/drupalchicago/">Drupal Chicago Meetup group</a></li>
        </ul>
```

**Review by froboy — COMMENTED** [review]
@kthull I added suggestions as per your comments.

**Review by kthull — APPROVED** [review]
ship it!

**Comment by froboy** [comment]
Yay! https://midwestopensourcealliance.org/

**Pull request merged**